src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```
@@ -1278,6 +1278,20 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ProjectCapability Include="CrossPlatformExecutable" />
   </ItemGroup>
 
+  <!-- Native AOT compilation -->
+  <ItemGroup>
+    <ProjectCapability Include="NativeAOTProperties" Condition="'$(ShowNativeAOTProperties)' == 'true'"/>
```

(nit) to be consistent with the ProjectCapability entries around it:

```suggestion
  <ItemGroup Condition="'$(ShowNativeAOTProperties)' == 'true'">
    <ProjectCapability Include="NativeAOTProperties" />
```